### PR TITLE
Add Spacy Italian pre-trained model into Rasa Docker image

### PR DESCRIPTION
**Pull request by ibaylyak**
**Proposed changes**:
- Add italian spacy model docker image 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by ibaylyak** [comment]
@aerowithanl What should I do to fix unsuccessful checks?

**Review by sanchariGr — APPROVED** [review]
@ibaylyak this looks good to me, can you please rebase against main.

**Comment by ibaylyak** [comment]
> @ibaylyak this looks good to me, can you please rebase against main.

done

**Comment by ibaylyak** [comment]
@sanchariGr do i have to do anything else?

**Review by rasa-jmac — APPROVED** [review]

**Comment by sanchariGr** [comment]
> @sanchariGr do I have to do anything else?

@ibaylyak Apologies for the late reply, unfortunately the building docker image failure issue that you are facing here is expected since you donot have the required permissions. We would need a bit longer to come up with a solution for you.

**Comment by ibaylyak** [comment]
> > @sanchariGr do I have to do anything else?
> 
> @ibaylyak Apologies for the late reply, unfortunately the building docker image failure issue that you are facing here is expected since you donot have the required permissions. We would need a bit longer to come up with a solution for you.

Ok

**Comment by rasa-jmac** [comment]
@ibaylyak I've just merged what I hope is a fix to main - let's see if this works for you now.

**Comment by ibaylyak** [comment]
> @ibaylyak I've just merged what I hope is a fix to main - let's see if this works for you now.

ok, thank you.

**Comment by rasa-jmac** [comment]
@ibaylyak We don't seem to be having very good luck with the CI checks for this PR. We're still investigating on our side and actively working on this.

**Comment by miraai** [comment]
@ibaylyak Hello, thank you for your patience on this, I kindly ask you to rebase against main once more and lets see if we have more luck this time :)

File: .github/workflows/continous-integration.yml [hunk 1]
```diff
@@ -750,7 +750,8 @@ jobs:
 
     strategy:
       matrix:
-        image: [default, full, mitie-en, spacy-de, spacy-en]
+        image: [default, full, mitie-en, spacy-it, spacy-de, spacy-en]
+        arch: [linux/amd64, linux/arm64]
```
[the remark this answers is not in the document]
Comment (ibaylyak) [reply .github/workflows/continous-integration.yml]: > Please remove `arch`, since we are not using it anymore :)

done

**Review by miraai — APPROVED** [review]

**Pull request merged**